include/z64.h

```diff
@@ -1774,4 +1774,15 @@ typedef struct {
 #define ROM_FILE_UNSET \
     { 0 }
 
+typedef enum {
+    /* 0 */ DEBUG_TEXT_YELLOW,
+    /* 1 */ DEBUG_TEXT_PEACH,
+    /* 2 */ DEBUG_TEXT_BROWN,
+    /* 3 */ DEBUG_TEXT_ORANGE,
+    /* 4 */ DEBUG_TEXT_GOLD,
+    /* 5 */ DEBUG_TEXT_WHITE,
+    /* 6 */ DEBUG_TEXT_BLUE,
+    /* 7 */ DEBUG_TEXT_GREEN
+} DebugTextColor;
```

Comment:
I could not think of a better place to put this. If there are any suggestions that dont involve making a new header right now, feel free

also, the colors in game:
![colors](https://user-images.githubusercontent.com/22045047/208368546-b9d4f8f4-730b-47b9-9974-d524109afc1f.png)

Reply:
Not the greatest color options. Feel like `peach` -> `red`, and although gold, orange are ok names, `gold` looks closer to orange and `orange` looks closer to mud.

Reply:
From https://rgbcolorcode.com/ :
laser lemon
light salmon
brown (traditional)
dark goldenrod
saffron
platinum
cornflower blue
chartreuse (web)

Reply:
I dont really agree on red, its not red enough 

the way orange/gold are used in debug camera, its often switched between with a ternary, implying that the darker "orange" is like an "off" version of the same color. But this is entirely speculation from just pasting the enum values in. Personally I think its fine like this